src/components/TransactionFlow/__tests__/ConfirmTransaction.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { simpleRender, fireEvent } from 'test-utils';
+
+import { fSettings, fTxConfig, fAccount } from '@fixtures';
+import { devContacts } from '@database/seed';
+import { ExtendedAddressBook, ITxType } from '@types';
+import { truncate } from '@utils';
+import { translateRaw } from '@translations';
+import { ZAPS_CONFIG } from '@features/DeFiZap/config';
+
+import { ConfirmTransactionUI, UIProps as Props } from '../ConfirmTransaction';
+import { constructSenderFromTxConfig } from '../helpers';
+
+const senderContact = Object.values(devContacts)[0] as ExtendedAddressBook;
+const recipientContact = Object.values(devContacts)[1] as ExtendedAddressBook;
+
+const defaultProps: Props = {
+  settings: fSettings,
+  txConfig: fTxConfig,
+  onComplete: jest.fn(),
+  assetRate: 250,
+  baseAssetRate: 250,
+  sender: constructSenderFromTxConfig(fTxConfig, [fAccount]),
+  senderContact,
+  recipientContact
+};
+
+function getComponent(props: Props) {
+  return simpleRender(<ConfirmTransactionUI {...props} />);
+}
+
+describe('ConfirmTransaction', () => {
+  test('it shows the addresses', async () => {
```

Comment:
The distinction is minor but `displays` sounds more natural than `shows`.
Maybe `contains`?